### PR TITLE
feat: add Marketing Analyst role to jobs page

### PR DESCRIPTION
**Pull request by marneysof**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/markdown/jobs/marketinganalyst.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Overview
+
+- **Position Type:** Full Time
+- **Location**: Munich, Germany
+- **Components**: Google analytics, Hubspot, CMS platforms
+
+### What You'll Do
+
+- Conceive and perform marketing analysis about the following topics: Ory customer journey, target groups & prospective customers, marketing efficiency (media efficiency, attribution modeling), competition
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:36]:
```suggestion
- Conceptualize and perform digital marketing analysis about the following topics: Ory customer journey (onsite), target groups & prospective customers, marketing efficiency (media efficiency, attribution modeling), competition
```

File: src/markdown/jobs/marketinganalyst.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Perform marketing reporting and data preparation. Define KPIs and create standardized / automated reports and dashboards for marketing and linking of different data sources to derive marketing strategies
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:37]:
```suggestion
- Perform marketing reporting and data preparation. Define KPIs (ROI, ARR, MRR) and create standardized/automated reports (Google Data Studios) and dashboards linking different data sources to derive marketing strategies
```

File: src/markdown/jobs/marketinganalyst.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+secure cloud platform. You will provide organizational support to Ory leadership
+to ensure smooth operations and communications. Together with the CEO, you will
+provide a core leadership function that collaborates with UX design,
+engineering, finance, marketing, and data analytics.
+
+### Overview
+
+- **Position Type:** Full Time
+- **Location**: Munich, Germany
+- **Components**: Google analytics, Hubspot, CMS platforms
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:32]:
```suggestion
- **Components**: Google Analytics (data Studios), Hubspot, CMS platforms
```

File: src/markdown/jobs/marketinganalyst.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+- Perform marketing reporting and data preparation. Define KPIs and create standardized / automated reports and dashboards for marketing and linking of different data sources to derive marketing strategies
+- Monitor and evaluate campaign performance, A/B testing experiments, developing insights on how to improve impact. This includes determining KPIs, testing hypotheses, forecasting campaigns, and communicating insights to relevant members of the organization. 
+- Keep track of data that will inform marketing strategy and budgeting. 
+- Communicate across departments (i.e. sales, engineering, HR) to keep them informed about relevant marketing performance data and analytics.
+
+### We’re looking for someone who has
+
+- A degree in economics, mathematics, business informatics, consumer insights or a similar field.
+- 5 + years of professional experience in data science, analytics, and marketing BI 
+- An understanding of data-driven marketing, customer journeys, traditional and digital marketing channels
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:46]:
```suggestion
- An understanding of data-driven marketing, customer journeys and digital marketing channels
```

File: src/markdown/jobs/marketinganalyst.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- Experience with marketing, analytics, reporting tools, and CMS platforms (i.e. Google analytics)
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:47]:
```suggestion
- Experience with marketing, analytics, reporting tools, and CMS platforms (i.e. Google Analytics, data Studios) 
```

File: src/markdown/jobs/marketinganalyst.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+- Keep track of data that will inform marketing strategy and budgeting. 
+- Communicate across departments (i.e. sales, engineering, HR) to keep them informed about relevant marketing performance data and analytics.
+
+### We’re looking for someone who has
+
+- A degree in economics, mathematics, business informatics, consumer insights or a similar field.
+- 5 + years of professional experience in data science, analytics, and marketing BI 
+- An understanding of data-driven marketing, customer journeys, traditional and digital marketing channels
+- Experience with marketing, analytics, reporting tools, and CMS platforms (i.e. Google analytics) 
+- Facility with G Suite tools and knowledge of SQL.
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:48]:
```suggestion
```

**Review by BlueAcidFrog — APPROVED** [review]

**Comment by aeneasr** [comment]
Good to go @BlueAcidFrog ?

File: src/markdown/jobs/marketinganalyst.mdx [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+- Conceptualize and perform digital marketing analysis about the following topics: Ory customer journey (onsite), target groups & prospective customers, marketing efficiency (media efficiency, attribution modeling), competition
+- Perform marketing reporting and data preparation. Define KPIs (ROI, ARR, MRR) and create standardized/automated reports (Google Data Studios) and dashboards linking different data sources to derive marketing strategies
+- Monitor and evaluate campaign performance, A/B testing experiments, developing insights on how to improve impact. This includes determining KPIs, testing hypotheses, forecasting campaigns, and communicating insights to relevant members of the organization. 
+- Keep track of data that will inform marketing strategy and budgeting. 
+- Communicate across departments (i.e. sales, engineering, HR) to keep them informed about relevant marketing performance data and analytics.
+
+### We’re looking for someone who has
+
+- A degree in economics, mathematics, business informatics, consumer insights or a similar field.
+- 5 + years of professional experience in data science, analytics, and marketing BI
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:45]:
```suggestion
- 4 + years of professional experience in data science, analytics, and marketing BI 
```

**Review by BlueAcidFrog — DISMISSED** [review]

File: src/markdown/jobs/marketinganalyst.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
+---
+published: true
+path: '/jobs/marketing-analyst/'
+title: 'Marketing Analyst'
+metaTitle: 'Open Marketing Careers at Ory'
+metaDescription: 'Read the Marketing Analyst job description.'
+lastUpdatedAt: 'December 12, 2021'
```
Comment (BlueAcidFrog) [inline src/markdown/jobs/marketinganalyst.mdx:7]:
```suggestion
lastUpdatedAt: 'December 17, 2021'
```

**Review by BlueAcidFrog — APPROVED** [review]

**Pull request merged**